streams/src/main/java/org/apache/kafka/streams/kstream/WindowedSerdes.java

```diff
@@ -54,6 +54,6 @@ public SessionWindowedSerde(final Serde<T> inner) {
      * Construct a {@code SessionWindowedSerde} object for the specified inner class type.
      */
     static public <T> Serde<Windowed<T>> sessionWindowedSerdeFrom(final Class<T> type) {
-        return new TimeWindowedSerde<>(Serdes.serdeFrom(type));
+        return new SessionWindowedSerde<>(Serdes.serdeFrom(type));
```

Comment:
What the heck? How did this this not break any tests? Do you mind adding a regression test of some kind?

Thanks for the catch!

Reply:
I've added a test in another PR for this. So we can save @kamalcph some time here.